src/Symfony/Component/HttpFoundation/File/MimeType/ExtensionGuesser.php

```diff
@@ -56,7 +56,7 @@ public static function getInstance()
     /**
      * Registers all natively provided extension guessers.
      */
-    private function __construct()
+    protected function __construct()
```

Comment:
@kalessil http://ocramius.github.io/blog/when-to-declare-classes-final/

The same goes for private to protected. When should you do it? When you need it. By opening up possibilities to extend something, you will have to maintain Backwards Compatibility. Unless there's a good reason to make this protected, such as a feature _in_ Symfony, I don't think this should be protected.